src/components/sbb-option-group/sbb-option-group.stories.jsx

```diff
@@ -0,0 +1,137 @@
+import { h } from 'jsx-dom';
+import readme from './readme.md';
+
+const label = {
+  control: {
+    type: 'text',
+  },
+  table: {
+    category: 'Option group',
+  },
+};
+
+const disabled = {
+  control: {
+    type: 'boolean',
+  },
+  table: {
+    category: 'Option group',
+  },
+};
+
+const iconName = {
+  control: {
+    type: 'text',
+  },
+  table: {
+    category: 'Option',
+  },
+};
+
+const value = {
+  control: {
+    type: 'text',
+  },
+  table: {
+    category: 'Option',
+  },
+};
+
+const disabledSingle = {
+  control: {
+    type: 'boolean',
+  },
+  table: {
+    category: 'Option',
+  },
+};
+
+const numberOfOptions = {
+  control: {
+    type: 'number',
+  },
+};
+
+const defaultArgTypes = {
+  label,
+  'icon-name': iconName,
+  value,
+  disabled,
+  disabledSingle,
+  numberOfOptions,
+};
+
+const defaultArgs = {
+  label: 'Option group',
+  'icon-name': undefined,
+  value: 'Option',
+  disabled: false,
+  disabledSingle: false,
+  numberOfOptions: 3,
+};
+
+const defaultDecorator = [
+  (Story) => (
+    <div style={'border: 3px solid red'}>
+      <Story />
+    </div>
+  ),
+];
+
+const createOptions = ({ value, numberOfOptions, disabledSingle, ...args }) =>
+  new Array(numberOfOptions).fill(null).map((_, i) => {
+    return (
+      <sbb-option {...args} value={`${value} ${i + 1}`} disabled={disabledSingle && i === 0}>
+        {`${value} ${i + 1}`}
+      </sbb-option>
+    );
+  });
+
+const Template = ({ label, disabled, ...args }) => [
+  <sbb-option-group label={label + ' 1'} disabled={disabled}>
+    {createOptions(args)}
+  </sbb-option-group>,
+  <sbb-option-group label={label + ' 2'} disabled={disabled}>
+    {createOptions(args)}
+  </sbb-option-group>,
+];
+
+const TemplateAutocomplete = (args) => {
+  return (
+    <sbb-form-field label="Autocomplete">
+      <input placeholder="Placeholder" />
+      <sbb-autocomplete>{Template(args)}</sbb-autocomplete>
+    </sbb-form-field>
+  );
+};
+
+export const Standalone = Template.bind({});
+Standalone.argTypes = { ...defaultArgTypes };
+Standalone.args = { ...defaultArgs };
+Standalone.decorators = defaultDecorator;
+
+export const Autocomplete = TemplateAutocomplete.bind({});
+Autocomplete.argTypes = { ...defaultArgTypes };
+Autocomplete.args = { ...defaultArgs };
+
+export default {
+  decorators: [
+    (Story) => (
+      <div style={'padding: 2rem'}>
+        <Story />
+      </div>
+    ),
+  ],
+  parameters: {
+    actions: {
+      handles: ['click'],
+    },
+    backgrounds: {
+      disable: true,
+    },
+    docs: {
+      extractComponentDescription: () => readme,
+    },
+  },
+  title: 'components/sbb-option-group',
```

Comment:
Honestly, I don't know :(

The only useful part of the stories of the option and the optgroup is the documentation because the usage is meaningless outside the autocomplete/select component
Therefore, for some time, I thought of putting them under "Internarls".

Might it make sense to have a quick discussion about this? @kyubisation @jeripeierSBB

Reply:
Move it in form fields